Components/typography.css

```diff
@@ -0,0 +1,31 @@
+h1{font-size: 3rem}
+
+h2{font-size: 2.5rem}
+
+h3{font-size: 2rem}
+
+h4{font-size: 1.5rem}
+
+h5{font-size: 1rem}
+
+h6{font-size: 0.5rem}
+
+.p4{font-size: 1.5rem}
+
+.p3{font-size: 1.3rem}
+
+.p2{font-size: 1rem}
+
+.p1{font-size: 0.8rem}
+
+.text-grey{color: var(--gray)}
+
+.text-primary{color: var(--secondary-hover)}
+
+.text-secondary{color: var(--secondary-color)}
+
+.text-st{text-align:left}
+
+.text-md{text-align: center}
+
+.text-ed{text-align: right}
```

Comment:
Name of these classes are not self-explanatory, it is unclear whether these classes are for size or alignment.
You can use `text-center`, `text-start`, `text-end`

Reply:
done. thanks for correcting me.